apps/logging/loggers.py

```diff
@@ -1,11 +1,22 @@
-import apps.logging.request_logger as logging
-
+from datetime import datetime
```

Comment:
START-CODE-FOR-REVIEW-SECTION

Adds logging items that are returned from `check_crosswalks()`,  `get_access_token_counts()` and  `get_grant_counts()` functions.